src/tiles/BackgroundTile.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <div class="tile background">
+        <div v-if="type === 'twitch'" id="twitch"></div>
+        <img v-else-if="type === 'image'" id="image" :src="this.src"/>
+    </div>
+</template>
+
+<script>
+    export default {
+        props: {
+            type: {
+                type: String,
+                required: true,
+            },
+            src: {
+                type: String,
+                required: true,
+            }
+        },
+
+        mounted() {
+            if(this.type === "twitch")
```

Comment:
Curly braces should be in the same line as instruction. Also, use apostrophes `'` instead of quotes `"` for strings